File: bidly-backend/src/main/java/com/bidly/bidly_backend/model/Seguro.java
package com.bidly.bidly_backend.model;

import jakarta.persistence.*;
import lombok.*;
import java.math.BigDecimal;

@Entity
@Table(name = "seguros")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Seguro {

    @Id
    @Column(name = "nropoliza")
    private String nroPoliza;

    private String compania;

    @Column(name = "polizacombinada")
    private String polizaCombinada;

    private BigDecimal importe;
}

File: bidly-fastapi/app/models/seguro.py
from sqlalchemy import Column, String, Numeric
from app.database import Base


class Seguro(Base):
    __tablename__ = "seguros"

    nropoliza       = Column(String, primary_key=True)
    compania        = Column(String)
    polizacombinada = Column(String)
    importe         = Column(Numeric(precision=12, scale=2))
